automated-testing/ab-testing/readme.md

```diff
@@ -0,0 +1,23 @@
+# ~Test type~
```

Comment:
Is there a reason for chosing strike through characters in the title ~Test type~ and later on in the document, it looks very odd - assuming this is because it should be replaced at some point with AB-testing in this case, why not write AB-testing at once?

Reply:
This was used as a placeholder to be changed later, I agree that it does not look good but it is meant to be changed. We can take out the strikethrough and replace it with something else?